docker/qiime_config

```diff
@@ -0,0 +1 @@
+temp_dir /data/scratch/
```

Comment:
As indicated in https://developer.basespace.illumina.com/docs/content/documentation/native-apps/spacedock-conventions#ScratchFolderforStoringFilesDuringAnalysis